py/client/tests/test_session.py

```diff
@@ -365,5 +364,41 @@ def test_publish_table(self):
                  sub_session2.fetch_table(shared_ticket)
 
 
```

Comment:
only one newline between not-top level things
```suggestion
```

Reply:
done